components/vendors/VendorForm.tsx

```diff
@@ -146,9 +147,9 @@ const VendorForm = ({ vendor, host, onSuccess, onCancel, isModal }: VendorFormPr
     <div>
       <div className="mb-3 flex justify-between text-xl font-bold">
         {vendor ? (
-          <FormattedMessage defaultMessage="Edit Vendor" />
+          <FormattedMessage id="vendor.edit" defaultMessage="Edit Vendor" />
         ) : (
-          <FormattedMessage defaultMessage="Create Vendor" />
+          <FormattedMessage id="vendor.create" defaultMessage="Create Vendor" />
```

Comment:
Adding `id`s will force translators to re-translate the strings. Is it necessary here?

Reply:
Good point, I'll just remove it elsewhere. I thought this was too new to have been translated, but I was wrong.

Reply:
These contributors are fast :-)